sdk/confidentialledger/Azure.ResourceManager.ConfidentialLedger/api/Azure.ResourceManager.ConfidentialLedger.netstandard2.0.cs

```diff
@@ -171,4 +218,67 @@ public partial class ConfidentialLedgerProperties
         public static bool operator !=(Azure.ResourceManager.ConfidentialLedger.Models.ConfidentialLedgerType left, Azure.ResourceManager.ConfidentialLedger.Models.ConfidentialLedgerType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class DeploymentType
```

Comment:
The type name is too generic, use `prepend-rp-prefix` to specify it.
Please refer to [hci](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md) for usage.